test/number-field.html

```diff
@@ -199,6 +199,40 @@
           expect(e.defaultPrevented).to.be.true;
           expect(numberField.value).to.equal('0');
         });
+
+        it('should increase value to min value when increaseButton is clicked', () => {
+          numberField.min = 2;
+
+          increaseButton.click();
+
+          expect(numberField.value).to.be.equal('2');
+        });
+
+        it('should increase value to min value with step when increaseButton is clicked', () => {
+          numberField.step = 0.1;
+          numberField.min = 0.5;
+
+          increaseButton.click();
+
+          expect(numberField.value).to.be.equal('0.5');
+        });
+
+        it('should decrease value to max value when decreaseButton is clicked', () => {
```

Comment:
Please also add a test confirming the following:

1. set `this.min` to a negative number, e.g. `-10`
2. make sure the "minus" control is disabled
3. make sure that once "plus" is pressed, number field has -10 as value (currently it will be 0)